sssd_test_framework/utils/sssctl.py

```diff
@@ -90,3 +93,28 @@ def passkey_register(
             )
 
         return result.stdout_lines[-1].strip()
+
+    def cache_expire(self, *args: str) -> SSHProcessResult:
+        """
+        Run ``sssctl cache-expire`` command.
+
+        :param args: Additional arguments.
+        :type args: str
+        """
+        return self.host.ssh.exec(["sssctl", "cache-expire", *args])
+
+    def cache_ls(self) -> SSHProcessResult:
```

Comment:
Why is this needed?

This is not part of sssctl command so it should not be here.

Reply:
It is probably not needed and even if, it shouldnt be there.